nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed_exclude-event-types_[6,).partial.md

```diff
@@ -1,5 +1,11 @@
 ### Exclude event types from auto subscribe
 
+If a message handler is defined for an event but no publishers information can be found the endpoint will log an error, as the following one, at startup:
```

Comment:
I'm not sure we need to document the log entry? Also that warning doesn't strictly belong into the `Exclude event types from auto subscribe` section imo.